js/apps/system/_admin/aardvark/APP/react/src/store/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { combineReducers, Reducer, applyMiddleware } from 'redux';
```

Comment:
I think Redux might actually be overkill for Aardvark but feel free to leave it in for now,

Reply:
Will discuss and potentially change later.